README.md

```diff
@@ -23,15 +23,28 @@ Either build the project or [download a compatible release version of the plugin
 The plugin is intended to support the [features and parameters specified in the SonarQube documentation](https://docs.sonarqube.org/latest/branches/overview/), with the following caveats
 * __Pull Requests:__ Analysis of Pull Requests is fully supported, but the decoration of pull requests is only currently available for Github, Gitlab and Bitbucket Server
 
-# Properties
+# Pull Request Decoration
+The settings for Pull Request decoration for the supported providers are available under Administration -> Pull Request. General configurations for all providers can be found in the SonarQube documentation for [Pull Request Decoration](https://docs.sonarqube.org/latest/analysis/pull-request/).
 
 ## Bitbucket Server
 To enable setting of several properties in SonarQube on project level is required.
 
 The property "projectKey" or "userSlug" are mandatory in order to decide which API endpoint should be used.
 
+## Github Enterprise
+Follow the setup as documented in the [SonarQube GitHub Enterprise Integration](https://docs.sonarqube.org/latest/instance-administration/github-application/) document.
+
+SonarQube scans can then be run with the following properties:
+
+| Key                                 | Example        | Description                                                |
+|-------------------------------------|----------------|------------------------------------------------------------|
+| sonar.pullrequest.branch            | feature-branch | Branch to run analysis on                                  |
+| sonar.pullrequest.key               | 123            | ID of the pull request                                     |
+| sonar.pullrequest.base              | master         | The base branch target of the pull request                 |
+| sonar.pullrequest.github.repository | my-org/my-repo | Full path to repository, including the GitHub organization |
```

Comment:
Did you try configuring the `sonar.pullrequest.github.repository` property in the UI? It's more convenient to do it from the command line, but it's not a requirement to do so if it's been configured on the project, and currently wont be supported in the 8.1 release.

Reply:
Will remove it if it won't be needed in the future.

Generally we try to avoid making any changes in the UI, and pass everything on the gradle.properties.